content/en/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -131,17 +131,14 @@ If you configured bindings by yourself, here are GCP permission requirements for
 
 #### Cluster setup to use Workload Identity for Full Kubeflow
 
-Public Kubeflow v1.0.1 release hasn't been configured out of box. [The fix](https://github.com/kubeflow/manifests/pull/969) has been merged, but not yet released publically.
-After the fix is released, if you deployed Kubeflow following the GCP instructions Workload Identity will have already been configured properly for Kubeflow Pipelines.
+Starting from Kubeflow 1.1, Kubeflow Pipelines [supports multi-user isolation](/docs/pipelines/multi-user/). Therefore, pipeline runs are executed in user namespaces using the `default-editor` KSA. The `default-editor` KSA is auto-bound to the GSA specified in user profile.
 
-If you want to use Workload Identity with pipelines on Kubeflow v1.0.1 or before, I recommend running the following commands to patch your deployment:
-```
-export NAMESPACE=kubeflow # Replace with your kubeflow's namespace if it's been customized.
-kubectl patch deployment -n ${NAMESPACE} ml-pipeline --patch '{"spec": {"template": {"spec": {"containers": [{"name": "ml-pipeline-api-server", "env": [{"name": "DEFAULTPIPELINERUNNERSERVICEACCOUNT", "value": "kf-user"}]}]}}}}'
-kubectl patch clusterrolebinding -n ${NAMESPACE} pipeline-runner --patch '{"subjects": [{"kind": "ServiceAccount", "name": "kf-user", "namespace": "'$NAMESPACE'"}]}'
-```
+You don't need to do any setup if you just want pipelines in your namespace
```

Comment:
@Bobgy I'd reverse the order here and remove "just" to show INTENT -> ACTION:

```
If you want pipelines in your namespace to use the default GSA in your profile,
you don't need to take extra setup steps.
```

This is also consistent with the next paragraph.

Reply:
INTENT -> ACTION

this is a mind blown strategy to me, thank you!